### PR TITLE
data: clean duplicates domain

### PR DESCRIPTION
**Pull request by akiirui**
data: clean duplicates domain
    
    Domain                  Exist In            Remove From
    71.am                   iqiyi               geolocation-cn
    88cdn.com               xunlei              geolocation-cn
    applecare.wang          apple               apple
    bmw-motorrad.ec         bmw                 bmw
    h2os.com                oneplus             oppo
    iciba.com               wps                 geolocation-cn
    idqqimg.com             tencent             geolocation-cn
    ijinshan.com            cheetahmobile       geolocation-cn
    kotlinlang.org          jetbrains           kotlin
    ministjohns.ca          mini                mini
    nikeit.com              nike                nike
    ntd.com                 epochmediagroup     ntd
    ntdtv.com               epochmediagroup     ntd
    paipai.com              jd                  geolocation-cn
    taylorandfrancis.com    taylorfrancis       taylorfrancis
    ximalaya.com            ximalaya            geolocation-cn
    xoyo.com                seasun              geolocation-cn
    yhd.com                 jd                  geolocation-cn
    yihaodianimg.com        jd                  geolocation-cn
    
    Extra:
    Domain                  Move To             From
    yihaodian.com           jd                  geolocation-cn

### CODE REVIEW

**Comment by Loyalsoldier** [comment]
Please rebase branch.

**Comment by Loyalsoldier** [comment]
Please remove empty list file and remove `include:emptylist`

File: data/kotlin [hunk 1]
```diff
@@ -1 +0,0 @@
-kotlinlang.org
```
Comment (Loyalsoldier) [inline data/kotlin:1]: Please remove the entire empty file and remove `include:kotlin`

File: data/ntd [hunk 2]
```diff
@@ -1,2 +0,0 @@
-ntd.com
-ntdtv.com
```
Comment (Loyalsoldier) [inline data/ntd:2]: Please remove the entire empty file and remove `include:ntd`

**Comment by akiirui** [comment]
> Please remove empty list file and remove `include:emptylist`

OMG, I forget these.
Done.

**Pull request merged**